Formula/hyper.rb

```diff
@@ -0,0 +1,23 @@
+class Hyper < Formula
+  desc "Hyper_(make VM run as container) client for https://www."
+  homepage "https://www.hyper.sh"
+  url "https://github.com/hyperhq/hypercli.git",
+    :tag => "v1.10",
```

Comment:
Yeah, the build process need the git commit to add into the version description, such as:
```
lei@leis-MBP:[~]$ hyper version
Client:
 Version:      1.10.0-beta
 API version:  1.23
 Go version:   go1.6.2
 Git commit:   603e437
 Built:        Mon Jul 11 03:09:34 UTC 2016
 OS/Arch:      darwin/amd64
```